### ISSUE TITLE
Map collision editor has stopped working on OSX

### ISSUE THREAD

**Issue by asmar1x** [bjorn/tiled#1293]
Version 0.16.1

This functionality was working before, but it has since stopped.  I'm hoping it is operator error and not a bug.  I have reinstalled multiple times and attempted to clear the application settings.

Selecting a tile and opening the collision editor works, but free-form lines inside the tile cannot be drawn.  Clicking in the tile puts the start of the line at the corner.  All shapes draw lines from corner to corner around the perimeter of the tile or across the tile, with the exception of ellipses which fill the entire tile.

I created a project on Windows with identical tiles and sizes, and the collision editor worked fine.

**Rectangles**
![screen shot 2016-06-04 at 3 18 44 pm](https://cloud.githubusercontent.com/assets/19752013/15801586/b45534e8-2a67-11e6-8829-be5a7faf350a.png)

**Circles**
![screen shot 2016-06-04 at 3 21 44 pm](https://cloud.githubusercontent.com/assets/19752013/15801593/1ba345fe-2a68-11e6-98fc-8a63b871032c.png)

**Triangles**
![screen shot 2016-06-04 at 3 13 09 pm](https://cloud.githubusercontent.com/assets/19752013/15801556/01d173a4-2a67-11e6-9f6c-9b108a5b8dd6.png)

**Polyline**
![screen shot 2016-06-04 at 3 14 50 pm](https://cloud.githubusercontent.com/assets/19752013/15801560/2398cf46-2a67-11e6-90cf-7d6ef21202f5.png)

**Comment by bjorn** [comment]
It seems like you turned on Snap to Grid?

**Comment by asmar1x** [comment]
I did, indeed.  I turned it off, and all works as before!  Definitely operator error!

**Comment by bjorn** [comment]
I'll close this then, but I will take it as a hint that usability is not the best here. I think it's probably confusing that this window is affected by that setting, though it is also a feature some people may rely on.

Hopefully it will be clearer once this extra window is merged into the main window, in the new tileset editor I'm working in the `wip/tilesetdocument` branch (which was mainly to make editing of external tilesets possible).
